plugins/postcss-stepped-value-functions/test/basic.css

```diff
@@ -0,0 +1,42 @@
+.mod {
+	padding: 8px mod(18px, 5px) 1px calc(mod(15px, 6px) + 50%);
+	transform: rotate(mod(-140deg, -90deg));
+}
+
+.rem {
+	padding: 8px rem(18px, 5px) 1px calc(rem(15px, 6px) + 50%);
+	transform: rotate(rem(-140deg, -90deg));
+}
+
+.mod-rem {
+	font-size: mod(-18px, 5px);
+	width: rem(-18px, 5px);
+	transform: rotateX(mod(140deg, -90deg)) rotateY(rem(140deg, -90deg));
+}
```

Comment:
Maybe some more complex tests?

```css
font-size: round(nearest, rem(mod(mod(-18px, 5px), 5px), 2px));
```

nesting same function, combining all of them, ...